classes/module/Module.php

```diff
@@ -1626,12 +1630,21 @@ public static function getModulesDirOnDisk()
      */
     public static function getNonNativeModuleList()
     {
-        return false;
+        $finder = new ContainerFinder(Context::getContext());
+        $sfContainer = $finder->getContainer();
+
```

Comment:
I suggest to add new method like `getModuleRepository` to avoid code duplicate in  **getNonNativeModuleList** and **getNativeModuleList**

Reply:
Fixed.